src/common/marlin_vars.h

```diff
@@ -150,6 +156,7 @@ typedef struct _marlin_vars_t {
     uint8_t media_inserted;      // media_is_inserted()
     uint8_t fan_check_enabled;   // fan_check [on/off]
     uint8_t fs_autoload_enabled; // fs_autoload [on/off]
+    float curr_pos[4];           // current position XYZE according to G-code [mm]
```

Comment:
4 floats does not have size 1B .. move it up
order is important - affects size

Reply:
👍